src/main/java/net/minecraftforge/event/world/ChunkDataEvent.java

```diff
@@ -64,9 +65,17 @@ public CompoundNBT getData()
      **/
     public static class Load extends ChunkDataEvent
     {
-        public Load(IChunk chunk, CompoundNBT data)
+        private ChunkStatus.Type status;
+
+        public Load(IChunk chunk, CompoundNBT data, ChunkStatus.Type status)
```

Comment:
This is a binary breaking change, please leave the old constructor and deprecate it

Reply:
what ChunkStatus.Type can I realistically default to in that case?

Reply:
I'd just set it to null, and mark `getStatus` as nullable. Mods shouldn't fire this event so it will be nonnull in 99% of the time, but it's better to make modders that use the new type account for a possible null then break old mods